assets/chezmoi.io/docs/reference/commands/unmanaged.md

```diff
@@ -5,11 +5,15 @@ unmanaged files in the destination directory.
 
 It is an error to supply *path*s that are not found on the filesystem.
 
-## `-p`, `--path-style` `absolute`|`relative`
+## `-p`, `--path-style` `absolute`|`relative`|`source-absolute`|`source-relative`
```

Comment:
This should *probably* be moved to `common.md` since it's used by `ignored`, `managed`, and `unmanaged`.

Reply:
Yeah, why not. In general I think common flags do not have strict definition and it's not clear which flags should be considered common.

Reply:
`common.md` defines it well enough:

> The following flags apply to multiple commands where they are relevant.

These would be manually defined, visible in `chezmoi <command> --help` as belonging to the command, but having the same options across multiple commands. Things like `--format` would be common *except* for the case where you’re determining the archive format.